_data/shaunferns26/shaunferns26.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shaun
+institution: BITS Pilani
+profile_pic: octocat.png
```

Comment:
Although that definitely looks like the suit you have and the shoes are pretty similar too, please use your own picture!